test/units/testsuite-70.service

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+[Unit]
+Description=TEST-70-LONG-UNIT-NAMES
```

Comment:
There's a runtime cost in setting up separate test cases, so it would be better to fold this test into an existing one

Reply:
Sure, there is a cost but isn't it wort paying for the gained clarity of test results you get by having separate test? I'd fully agree with you if we had a test framework where you could clearly denote that this is separate test case and it's result would be clearly visible and distinct from all other test cases, but we don't have that.

At any rate, I am happy to fold this into some existing test if you think that what I wrote above is a trade-off we want (and to some degree encourage).

Reply:
it's worth it when there's a clearly separate component, or a clearly different setup - but there's plenty of tests already running checks on services and units

Reply:
Any idea which test should we merge this into? Maybe make it part of the storage test-suite in `TEST-64-UDEV-STORAGE`?

Reply:
TEST-60-MOUNT-RATELIMIT already checks other corner cases of mount units, so I'd add it there - don't worry that it doesn't match the name, I have a TODO to merge lots of these and rename them